examples/misc/lib/tutorial/cat_no_hash.dart

```diff
@@ -7,7 +7,7 @@ Future<void> main(List<String> args) async {
   var lines = file
       .openRead()
       .transform(utf8.decoder) //!<br>
-      .transform(const LineSplitter());
+      .transform(LineSplitter());
```

Comment:
Is this really optional here?

Reply:
It isn't optional in the sense that the context isn't a const context, but the const qualifier isn't functionally necessary.